source/renderer/app/stores/SidebarStore.js

```diff
@@ -84,7 +84,7 @@ export default class SidebarStore extends Store {
       [categories.STAKING_DELEGATION_COUNTDOWN.name]: false,
       [categories.STAKING.name]: true,
       [categories.SETTINGS.name]: true,
-      [categories.VOTING.name]: isMainnet || isDev,
+      [categories.VOTING.name]: isMainnet || isDev || isTestnet,
```

Comment:
Please remove this. You are in dev env so you will have voting enabled.

Reply:
After QA team confirms that its working as expected i will revert it

Reply:
Please add this to TODO section then 🙏